packages/devtron-fe-common-lib/.eslintignore

```diff
@@ -0,0 +1,69 @@
+.eslintrc.cjs
```

Comment:
- Move eslint to the top level along with these files
- What about linting for fe-lib

Reply:
Will be doing it with your guidance :)